src/nbl/ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -1621,7 +1607,11 @@ inline core::smart_refctd_ptr<asset::ICPUDescriptorSet> CMitsubaLoader::createDS
 		auto* meta = static_cast<const IMeshMetadata*>(mesh->getMetadata());
 		
 		core::vectorSIMDf emissive;
-		uint32_t instDataOffset = instanceData.size();
+		for (uint32_t i = 0u; i < mesh->getMeshBufferCount(); ++i)
+		{
+			auto* mb = mesh->getMeshBuffer(i);
+			mb->setBaseInstance(instanceData.size());
+		}
 		for (const auto& inst : meta->getInstances()) {
 			// @Crisspl IIRC lights in mitsuba have "sides" , TODO
```

Comment:
nah, area emitter doesnt have anything like that

Reply:
there's no explicit flag etc. but if you look at the way their lights work... they have a side
![image](https://user-images.githubusercontent.com/6894321/104245390-761ce500-5464-11eb-88b6-d2a1d33a23fb.png)

Reply:
hmm, interestimg, nothing about this in the docs...

Reply:
to fix